### PR TITLE
[docs] Update upgrading-metabase.md

### PR DESCRIPTION
**Pull request by likeshumidity**
added note about major version upgrades taking longer (6-8 weeks) and instructions on requesting early upgrade

### CODE REVIEW

File: docs/installation-and-operation/upgrading-metabase.md
Comment (jeff-bruemmer) [inline docs/installation-and-operation/upgrading-metabase.md:132]:
```suggestion
If you're on a [Metabase Cloud](https://www.metabase.com/pricing) plan, we'll upgrade your Metabase automatically with each new release; no action needed on your end. How soon we upgrade you depends on the the type of release:

- Minor releases (e.g., x.47.4 to x.47.5): Usually about a week.
- Major releases (e.g., x.47.4 to x.48.0): Longer, usually weeks (just to make sure everything goes smoothly).

Cloud customers can request an early upgrade by emailing support at help@metabase.com. Include the URL of the Metabase you want us to upgrade.
```

This edit is easier to skim, and doesn't put a hard bound on major upgrade timing.

**Review by jeff-bruemmer — DISMISSED** [review]

**Comment by likeshumidity** [comment]
@jeff-bruemmer sorry can you approve again? I missed the window after approval to merge.

**Review by jeff-bruemmer — APPROVED** [review]

**Pull request merged**
